.gitlab-ci.yml

```diff
@@ -78,6 +78,17 @@ test-linux-stable:                 &test
     - time cargo test --all --release --verbose --locked
     - sccache -s
 
+check-web-wasm:
+  stage:                           test
+  image:                           tomaka/cargo-web:latest
+  allow_failure: true
```

Comment:
I put an `allow_failure: true` in order to be conservative and not accidentally block PRs in case something bad happens, however please note that the whole point of this CI step is to detect when the build breaks.

Reply:
I don't see the `check-web-wasm` test on github though, and I'm not sure how it works to add it.

Reply:
> I don't see the `check-web-wasm` test on github though, and I'm not sure how it works to add it.

you would possibly want this for pull requests - for that the `only: master` statement has to be changed.